model_compression_toolkit/exporter/model_exporter/pytorch/fakely_quant_pth_pytorch_exporter.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Callable
+
+import torch.nn
+
+from model_compression_toolkit.core.common import Logger
+from model_compression_toolkit.core.pytorch.utils import to_torch_tensor
+from model_compression_toolkit.exporter.model_exporter.pytorch.base_pytorch_exporter import BasePyTorchExporter
+
+
+class FakelyQuantPTHPyTorchExporter(BasePyTorchExporter):
```

Comment:
Why do you call it PTH ?  It's TorchScript.  The extension is .pth but we call it TorchScript format. .pth can be also pure pytorch.